snabble/Android-SDK
# Apply 3ds changes for telecash (Apps-1687)

To add payment credentials for fiserv it is now required to additional data. To provide these information the user now has to enter the required fields first and is then navigated to the old input mask.

APPS-1687

### How to test?
Launch the application and switch to the testing env. 
Add a payment method and see the new flow.

### Definition of Done


#### App Tests

#### Testing
